tools/acrn-crashlog/acrnprobe/conf.rst

```diff
@@ -0,0 +1,379 @@
+.. _acrnprobe-conf:
+
+Configuration Of Acrnprobe
+##########################
+
+Description
+***********
+``Acrnprobe`` uses XML as the format of its configuration file, namely
+``acrnprobe.xml``, so the configuration file should reach the `XML standard`_
+at least.
+
+Layout
+******
+
+.. code-block:: xml
+
+   <?xml version="1.0" encoding="UTF-8"?>
+   <conf>
+   Root node of configuration.
+
+   <senders>
+   Configuration section of senders.
+   </senders>
+
+   <triggers>
+   Configuration section of triggers.
+   </triggers>
+
+   <vms>
+   Configuration section of virtual machines.
+   </vms>
+
+   <logs>
+   Configuration section of logs.
+   </logs>
+
+   <crashes>
+   Configuration section of crashes.
+   Note that this section should be configured after triggers and logs, as
+   crashes depend on these two sections.
+   </crashes>
+
+   <infos>
+   Configuration section of infos.
+   Note that this section should be configured after triggers and logs, as
+   infos depend on these two sections.
+   </infos>
+
+   </conf>
+
+As for the definition of ``sender``, ``trigger``, ``crash`` and ``info`` and
+information of supported ``vm``, please refer to :ref:`acrnprobe_doc`.
+
+Properties of modules
+*********************
+
+Common properties
+=================
+
+- ``id``:
+  The index, which grows from 1, in its group.
+- ``enable``:
+  This part of configuration can only take effect when the value is ``true``.
+
+Other properties
+================
+
+- ``inherit``:
+  Specify a parent for a certain crash.
```

Comment:
For clarity, write this as:

    Specify a parent for a certain crash configuration.